src/components/HomePage.tsx

```diff
@@ -0,0 +1,3 @@
+export const HomePage: React.FC = () => (
+  <h1 className="title">Home Page</h1>
+);
```

Comment:
probably for such small components it will be redundant
React will try to keep on track this small component, and it won't cause more optimization comparing with it's rendering